src/main/java/sirius/db/mixing/properties/BaseEntityRefProperty.java

```diff
@@ -142,8 +142,16 @@ protected Object transformToDatasource(Class<? extends BaseMapper<?, ?, ?>> mapp
      */
     protected abstract Optional<E> find(Class<E> type, Value value);
 
+    /**
+     * Resolve the referenced EntityRef
```

Comment:
The referenced entity reference? Please clarify 😅

```suggestion
     * Resolves the referenced entity.
```